data/xml/2019.icon.xml

```diff
@@ -0,0 +1,283 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2019.icon">
+  <volume id="1" ingest-date="2021-05-10">
+    <meta>
+      <booktitle>Proceedings of the Sixteenth International Conference on Natural Language Processing</booktitle>
```

Comment:
All [their other proceedings](https://aclanthology.org/venues/icon/) used numbers (e.g., 16th), I think we should change it.